game_board.py

```diff
@@ -39,20 +47,45 @@ def map_cord(str_board):
             map_xy.append(char_xy)
             x += 1
         y += 1
+    
     return map_xy
 
-def move_plus():
+def make_plus_list():
     """
     Finds all the "+" chars in the map_xy 
     Returns a list of lists where every list has ["+",xPos,Ypos]
     Example [['X', 0, 0], [' ', 1, 0],...,['+', '4', '0']] => [['+', '4', '0'],...]
     """
-    map_xy =map_cord(read_map(map_path))
-    move = []
+    map_xy =convert_map_to_coordinates(read_map(map_path))
+    plus_list = []
     for cord in map_xy:
         if cord[0] == "+":
-            move.append([cord[0],str(cord[1]),str(cord[2])])
-    return move
+            plus_list.append([cord[0],str(cord[1]),str(cord[2])])
+    return plus_list
+
+def print_player_names(screen,player1_name,player2_name):
+    """Prints the playernames in screen 
+
+    Keyword arguments:
+    screen -- the curses screen
+    player1_name -- player1 name as string
+    player2_name -- player1 name as string
+    """
+    curses.init_pair(3,curses.COLOR_RED,curses.COLOR_BLACK)
```

Comment:
You don't have to run init_pair everytime this function is called. Move this creation of the color pair to the main function and then you can just use `curses.color_pair(3)` as you are doing at the moment. Just make sure that two color pairs aren't created with the same number